### ISSUE TITLE
Install-chromedriver step fails without a clear error message.

### ISSUE THREAD

**Issue by jonathan-flowers** [CircleCI-Public/browser-tools-orb#15]
### Orb version

1.0.1

### What happened

`install-chromedriver` step fails without a clear error message.

The `install-chrome` step works as expected, but when running `install-chromedriver`, I receive:

```
Installed version of Google Chrome is 84.0.4147.135
ChromeDriver 84.0.4147.30 will be installed

Exited with code exit status 1
```

I am using the `circleci/node:10.13` image as my Docker executor.

I can't tell why this is failing or how to fix it.

### Expected behavior

I would assume that this would install chromedriver without an unexpected failure

**Comment by dsayling** [comment]
@jonathan-flowers I tried reproducing your described failure with the following config: 
```
version: 2.1
orbs:
  browser-tools: circleci/browser-tools@1.0.1

jobs: 
  sample:
    docker:
      - image: circleci/node:10.13
    steps:
      - browser-tools/install-chrome
      - browser-tools/install-chromedriver
      - run:
          command: |
            chromedriver --version


workflows:
  sample_wf: 
    jobs: 
      - sample
```

Could you provide any further details?

**Comment by jonathan-flowers** [comment]
Hi @dsayling ,
Thanks for getting back to me!  The error itself was very strange; I took a  break away from the project because I couldn't get it working. However I just looped back to it to check and it doesn't look like I can reproduce it either. May have just been a temporary hiccup somewhere or maybe some sort of weird mistake or missing component on my part.

Thanks for checking it out anyways! Sorry about that. I will close this.
